src/bin/pgcopydb/string_utils.c

```diff
@@ -701,11 +701,11 @@ pretty_print_bytes(char *buffer, size_t size, uint64_t bytes)
 		"PB",                   /* Peta */
 		"EB"                    /* Exa */
 	};
-
+	uint suffixes_len = sizeof(suffixes) / sizeof(suffixes[0]);
 	uint sIndex = 0;
 	long double count = bytes;
 
-	while (count >= 10240 && sIndex < 7)
+	while (count >= 10240 && sIndex < suffixes_len - 1)
```

Comment:
Can we parenthesize `(suffixes_len - 1)` please? I hate to have to remember the specifics of the operation precedence when reviewing code. Maybe I'm too used to Lisp which does not need any precedence rules.

Reply:
Sure, changed.